docker-compose.yml

```
@@ -5,7 +5,8 @@ volumes:
     driver: juicedata/juicefs
     driver_opts:
       name: polyfill-cache
-      metaurl: sqlite3://polyfill-cache.db
+      # metaurl: sqlite3://polyfill-cache.db
+      metaurl: postgres://postgres:${ADMIN_JWT_SECRET}@postgres-server:5432/juicefs
```

```suggestion
      metaurl: postgres://postgres:${META DATA_PASSWORD}@postgres-server:5432/juicefs
```
以前的密码变量我只是借用所在项目中现有的密钥字串，这个项目不需要拘泥于以前的命名。